test/e2e_main_test.ts

```diff
@@ -18,7 +18,7 @@ describe('toClosureJS', () => {
     const closure = toClosureJS(
         {sourceMap: true, experimentalDecorators: true} as ts.CompilerOptions,
         ['test_files/underscore/export_underscore.ts', 'test_files/underscore/underscore.ts'],
-        {isUntyped: false} as Settings, diagnostics);
```

Comment:
I don't think so.  Sometimes using "as" is legit.  I blame your reviewer (probably me). :P 

I wish they had made the syntax for coercions a bit heavier so it'd stand out more.  Like in C++ you have to write `reinterpret_cast<Settings>(...)` and it's more obvious at review time that it's not what you want except in exceptional scenarios.